pkg/datastore/helper.go

```diff
@@ -207,3 +209,38 @@ func openDb(l logger.Interface, dbHost string, dbPort int, dbUsername, dbPasswor
 
 	return db, nil
 }
+
+// Create a Postgres DB using the provided config if it doesn't exist.
+func DBCreate(cfg DBConfig) error {
+	if DBExists(cfg) {
+		return nil
+	}
+	dataSourceName := fmt.Sprintf("host=%s port=%d user=%s password=%s sslmode=%s",
+		cfg.host, cfg.port, cfg.username, cfg.password, cfg.sslMode)
+	db, err := gorm.Open(postgres.Open(dataSourceName), &gorm.Config{})
+	if err != nil {
+		return err
+	}
+
+	createDbCmd := fmt.Sprintf("CREATE DATABASE %s;", cfg.dbName)
```

Comment:
All of the SQL commands we're constructing manually and executing are separate functions (e.g., `getDropTriggerStmt`, `getTruncateTableStmt`, etc.). For consistency, I propose you create `getCreateDatabase` function and move it to sql_struct.go

Reply:
noted these are for test only, skipping